gp-templates/helper-functions.php

```diff
@@ -287,6 +287,95 @@ function textareas( $entry, $permissions, $index = 0 ) {
 	<?php
 }
 
+/**
+ * Render multiple notes
+ *
+ * @param GP_Translation $entry            The translation entry.
+ * @param GP_Glossary    $permissions      Permissions of the user.
+ *
+ * @return void
+ */
+function render_notes( $entry, $permissions ) {
+	list( $can_edit, $can_approve ) = $permissions;
+	$notes                          = GP::$notes->get_by_entry( $entry );
+?>
+	<dl>
+		<dt>
+			<?php echo esc_attr__( 'Action Log:', 'glotpress' ); ?>
```

Comment:
We should make this consistent in the UI, at the moment it's "Action Log" and "New Reviewer Note", perhaps "Notes" and "New note"?